README.md

```diff
@@ -58,7 +62,11 @@ Multi-signature transactions are accommodated under-the-hood about 80%, and will
  Windows: [Download](http://www.py2exe.org/)  
 
 * LMDB - database engine, modified to suit Armory's use cases
-[LMDB page](http://symas.com/mdb/)
+ Linux:   Install package `liblmdb-dev` and `python-lmdb`
```

Comment:
These packages are provided as part of the source code here and have custom modifcation.

Reply:
Hmm. In `./cppForSwig/lmdb` I guess?  Then the README should not mention lmdb as a dependency, correct?

Reply:
IMO it should be listed as a dependency without a link and a note that it is provided as part of the source code. Same with cryptopp above.

Reply:
Done.